scripts/signing_kernel_modules.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+# This script is signing kernel modules by using sign-file tool 
+
+usage() {
+    cat <<EOF
+$0: # Display Help
+$0 <LINUX_KERNEL_VERSION> <PEM_CERT> <PEM_PRIVATE_KEY> <LOCAL_SIGN_FILE> <LOCAL_EXTRACT_CERT> <KERNEL_MODULES_DIR>
+Sign kernel modules in <KERNEL_MODULES_DIR> using private & public keys.
```

Comment:
i suggest to use getopt to define those option clearly instead of just providing all these raw arugments

Reply:
done